custom/champ/views.py

```diff
@@ -78,7 +78,7 @@ def get_target_data(self, domain):
             clienttype[idx] = type.lower()
 
         config.update({'clienttype': clienttype})
-        target_data = TargetsDataSource(config=config).data
+        target_data = list(TargetsDataSource(config=config).data.values())[0]
```

Comment:
what was the process used to determine that we caught all possible instances of something like this?

Reply:
Other than a thorough code search we've also had a soft assert in place for some time now which has only ever triggered on UCRs (which is addressed in this PR)

Reply:
Plus tests(if they were available for the custom data source/report)